api/types/src/convert.rs

```diff
@@ -228,7 +237,11 @@ impl<'a, S: StateView> MoveConverter<'a, S> {
                     },
                 })
             },
-            ValidatorTransaction(_txn) => (info, events, timestamp).into(),
+            aptos_types::transaction::Transaction::ValidatorTransaction(txn) => {
+                Transaction::ValidatorTransaction(ValidatorTransaction::from_internal_repr(
```

Comment:
Generally I would have taken the approach of special casing the single one rather than changing the rest of the match statement

Reply:
fixed